src/test/java/com/google/api/codegen/testdata/simplecompute_gapic.yaml

```diff
@@ -205,12 +205,10 @@ interfaces:
       groups:
       - parameters:
         - address
-        - requestId
```

Comment:
The `requestId` params were removed from the config baseline several PRs ago, so they are removed here in the actual config to reflect those changes.